sycl/CMakeLists.txt

```diff
@@ -190,7 +193,6 @@ add_subdirectory( source )
 # SYCL toolchain builds all components: compiler, libraries, headers, etc.
 add_custom_target( sycl-toolchain
   DEPENDS ${SYCL_RT_LIBS}
-          pi_opencl
```

Comment:
Why removing OpenCL support?

Reply:
The pi_opencl dependency is added to sycl-toolchain in the opencl plugin cmake `sycl/plugins/opencl/CMakeLists.txt` in https://github.com/intel/llvm/commit/884459b63f44239f8e6453dc3f9ed0f2502b8fc3#diff-edef34e019c51a57812a69f7a111afdcR22